addons/memes.py

```diff
@@ -37,7 +37,19 @@ def __init__(self, bot):
     async def s_99(self, ctx):
         """Memes."""
         await self._meme(ctx, "**ALL HAIL BRITANNIA!**")
+        
+    @commands.command(pass_context=True, hidden=True)
+    @commands.cooldown(rate=1, per=15.0, type=commands.BucketType.channel)
+    async def warm(self, ctx):
+        """Memes."""
+        await self._meme(ctx, "User has been warmed.")
 
+    @commands.command(pass_context=True, hidden=True)
+    @commands.cooldown(rate=1, per=15.0, type=commands.BucketType.channel)
+    async def bean(self, ctx):
+        """Memes."""
+        await self._meme(ctx, ":bean:")
```

Comment:
This will not work for the bot; correct syntax should be (for animated emote): `<a:bean:462076812076384257>`

Reply:
thank you, this has been updated